### PR TITLE
S2U-30 5.1.4.1 Assignments: When an assignment is created, assign it by default to the groups where the user is an instructor - Master

### PR DESCRIPTION
**Pull request by JuanDavid102**
If the user it is related to one or more groups, set it by default when creating a new task:
imagen
But if it isn't related to any group:
imagen

### CODE REVIEW

File: assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm [hunk 1]
```diff
@@ -53,8 +53,8 @@ can be either all members of the site, all members of selected groups, or the se
 			#set($hideGB = "if (document.getElementById('addToGradebook') != null){document.getElementById('addToGradebook').style.display = 'none';document.getElementById('$!name_Addtogradebook').checked=false};")
 		#end
 		<input type="radio" name="$name_AssignTo" id="groups" value="$value_AssignTo_IndividualsFromGroups"
-			onclick="$hideGB$('#groupTableContainer').fadeIn('slow'); ASN_INE.validateGroupSelection();ASN.resizeFrame(); if($!isTimesheet){ASN.enableTimesheetSetupSection();}"
-			#if( $value_AssignTo == $value_AssignTo_IndividualsFromGroups && !$selectedGroupsNotFound ) checked="checked"
+			onclick="$hideGB$('#groupTableContainer').fadeIn('slow'); ASN_INE.validateGroupSelection();ASN_INE.handleAssignToChangeForPeerAssessment();ASN.resizeFrame(); if($!isTimesheet){ASN.enableTimesheetSetupSection();}"
```
Comment (mpellicer) [inline assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm:56]: ASN_INE.handleAssignToChangeForPeerAssessment()  Where does this come from?

Comment (JuanDavid102) [reply assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm:56]: I don't know where i think that is an older function, i will remove it

Comment (mpellicer) [reply assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm]: Please, could you test this without that ? I don't think it belongs to the functionality.

Comment (JuanDavid102) [reply assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm]: I have tested it and works without that

**Review by mpellicer — APPROVED** [review]

**Pull request merged**